### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by leighlars**
#### What's this PR do?
This fulfills the basic login functionality, tests, styling for login and logout. 
#### Where should the reviewer start?
Review Login.js and respective files

#### How should this be manually tested?
Review Login.test.js

#### Any background context you want to provide?
There are opportunities to refactor. I also think we could investigate an Error functional component?

#### What are the relevant tickets? 
#24

#### Screenshots (if appropriate) 
N/a

#### Questions: 
N/a

### CODE REVIEW

File: src/Nav/Nav.js [hunk 1]
```diff
@@ -14,12 +14,14 @@ const Nav = (props) => {
       </button> 
       <button 
         className={props.isLoggedIn ? 'hidden' : ''}
-        id='login-btn'>
+        id='login-btn'
+        onClick={props.pageView}>
```
Comment (GreysonElkins) [inline src/Nav/Nav.js:18]: Something got mixed up here and the `showLoginPage` function isn't getting passed. I've got a fixed copy already, let me know if you'd like me to push it to the branch before we merge.

Comment (leighlars) [reply src/Nav/Nav.js:18]: Yes go for it!

File: src/App/App.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react'
 import './App.scss'
 import Header from '../Header/Header'
```
[the remark this answers is not in the document]
Comment (leighlars) [reply src/App/App.js:3]: I agree

File: src/Header/Header.js [hunk 3]
```diff
@@ -3,28 +3,20 @@ import Search from '../Search/Search'
...
 import './Header.scss'
 
-class Header extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {}
-  }
-
-  render() {
+const Header = (props) => {
```
Comment (leighlars) [inline src/Header/Header.js:6]: This makes so much more sense

File: src/Header/Header.js [hunk 4]
```diff
@@ -17,6 +17,7 @@ class Header extends Component {
         <Nav 
           isLoggedIn={this.props.isLoggedIn} 
           showLoginPage={this.props.showLoginPage}
+          showHomePage={this.props.showHomePage}
```
Comment (leighlars) [inline src/Header/Header.js]: Nice!

**Pull request merged**